compiler/tests/fail/stack_allocation/x86-64/writage_reg_ptr_args_not_disjoint.jazz

```diff
@@ -0,0 +1,16 @@
+fn f (reg ptr u64[1] r1, reg ptr u64[1] r2) -> reg ptr u64[1] {
+  r1[0] = r2[0];
```

Comment:
Can you introduce a intermediate variable to be sure that the error will not come from the body of f.

Reply:
I did it.